### ISSUE TITLE
Menu doesn't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jennorby/mupen64plus#344]
```
Describe your system:
 - Operating System (be specific): Mac OSX 10.5.8
 - Machine type (32-bit or 64-bit): 32 bit
 - Mupen64Plus version: 1.99.3
 - Plugins used: Glide

For visual artifacts, give:
- game name and CRC/MD5 hash (printed on console)
- description of the artifact (missing polygons, flashing, wrong colors,
etc)

Describe the problem:
When I try to select items in the menu, I am unable to.  Using the control 
stick only lets me cycle 
through the different menu screens (items, world map, etc)

Please provide any additional information below.

```

Original issue reported on code.google.com by `mrku...@yahoo.com` on 3 Mar 2010 at 1:38

**Comment by GoogleCodeExporter** [comment]
```
Which menu?
If you mean a specific game then please specify which you mean. And glide64 
wasnt
released of 1.99.3 yet. So please also add the revision of Glide. What Input 
are you
using? Have you checked what buttons are assigned?
```

Original comment by `sven@narfation.org` on 3 Mar 2010 at 9:13

**Comment by GoogleCodeExporter** [comment]
Original comment by `richard...@gmail.com` on 1 Dec 2010 at 1:30
- Changed state: **Invalid**
